docs/vscode.md

```diff
@@ -3,50 +3,75 @@
+-   [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome)
+-   [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+-   [yarn](https://marketplace.visualstudio.com/items?itemName=gamunu.vscode-yarn)
+-   [vscode-icons](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons)
+-   [export-index](https://marketplace.visualstudio.com/items?itemName=BrunoLM.export-index)
 
 The ESLint and Prettier integrations can be configured to fix and format your code on save.
 The Python integration can be configured to use Black as a formatter and run checks with Pylint,
 whether they are globally installed or in a virtualenv. Sample `.vscode/settings.json`:
 
 ```json
 {
-  "[typescript]": {
-    "editor.formatOnSave": true
-  },
-  "[typescriptreact]": {
-    "editor.formatOnSave": true
-  },
-  "[python]": {
-    "editor.formatOnSave": true
-  },
-  "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.organizeImports": false
-  },
-  "python.testing.pytestArgs": [
-    "flask"
-  ],
-  "python.testing.unittestEnabled": false,
-  "python.testing.nosetestsEnabled": false,
-  "python.testing.pytestEnabled": true,
-  "python.formatting.provider": "black",
-  "python.formatting.blackPath": "flask/venv/bin/black",
-  "python.linting.pylintEnabled": true,
-  "python.linting.enabled": true
+    "[typescript]": {
+        "editor.formatOnSave": true
+    },
+    "[typescriptreact]": {
+        "editor.formatOnSave": true
+    },
+    "[python]": {
+        "editor.formatOnSave": true
+    },
+    "editor.codeActionsOnSave": {
+        "source.fixAll.eslint": true,
+        "source.organizeImports": false
+    },
+    "python.testing.pytestArgs": ["flask"],
+    "python.testing.unittestEnabled": false,
+    "python.testing.nosetestsEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.formatting.provider": "black",
+    "python.formatting.blackPath": "flask/venv/bin/black",
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true
 }
 ```
+
+## Debugging in the Flask container
+
+-   [Debugging in VSCode](https://code.visualstudio.com/docs/editor/debugging)
+-   [Remote - Containers extension](https://code.visualstudio.com/docs/remote/containers)
+-   [Using profiles with Compose](https://docs.docker.com/compose/profiles/)
+
+To directly enter a Docker development that supports debugging in the container and launch the debugger:
+
+1. `docker-compose up -d app_debug`
+2. Click on remote connection icon in VSCode, choose "Attach to running container".
+3. When the prompt "Select the container to the attach VS Code" comes up, choose "/stager-app_debug_1".
+4. In the VSCode window that runs in the app container, use F5 to launch the debugger.
+
+To start the Docker development without debugging functionality enabled, and later enable the debugger functionality:
+
+1. `docker-compose up`
```

Comment:
I think we can take out this first step, it's not actually part of the procedure.